capabilities/system-integrations.html

```diff
@@ -0,0 +1,31 @@
+---
+layout: capability
+
+asset-name: system_integrations
+
+description: "We’re good at moving data around, safely. See our capabilities with system
+integration and how we can help your business."
+
+tags: [System Integration]
+
+title: System Integrations
+
+subtitle: "Taking scattered information and organizing it so it’s easy for you to use."
+
+text: '<p>We have had several clients who needed system integrations support. They
+typically had an application with a database but later there’s a need to integrate with a
+third-party piece of software, such as a registration order system or payment portal, and
+they need access to the data. We have set up numerous APIs to work off their existing
+database. Easy for us, and the customer only sees the end result – a nightmare avoided. </p>
+<p>Another example would be if somebody came to us and said, “I have this application and
+it’s already been built; it’s been running and now I want to take payments on it.” We have
+experience taking payments for all sorts of things, and then integrating with a bunch of
+different merchants. We have had clients whose business takes payments, but their system
+isn’t PCI compliant – that compliance can be expensive. Instead, we set them up with a
+payment merchant (like authorize.Net, or PayPros) that will take the credit card details.
+Our client only stores these things called tokens in their system, which can only be used
+by that system.</p>
```

Comment:
This sentence needs to be discussed. I think we need to be cautious around statements about tokens as they are  prone to security vulnerabilities. 
Maybe more details about how the tokens are used to interface with the payment system could help.

Reply:
Yes

Reply:
Fixed.